PerfCounterReporter/CounterDefinitions/system.counters

```diff
@@ -80,4 +80,7 @@
 #bottleneck. This counter is a sum of �Network Interface\\Bytes Received/sec� and 
 #�Network Interface\\Bytes Sent/sec�.  In some situations, you may wish to determine 
 #both inbound and outbound network traffic separately
-\Network Interface(*)\Bytes Total/sec
\ No newline at end of file
+\Network Interface(*)\Bytes Total/sec
+
+\LogicalDisk(*)\Free Megabytes
+\LogicalDisk(*)\% Free Space
```

Comment:
Do we also get the number of bytes free on a disk so that a user can get a sense of how much space is actually left without having to know the size of the disk out of band?